src/fastoad/io/configuration/configuration.py

```diff
@@ -249,6 +271,35 @@ def set_optimization_definition(self, optimization_definition: Dict):
         subpart = {"optimization": subpart}
         self._data.update(subpart)
 
+    def _configure_driver(self, prob):
+        driver_config = self._data.get(KEY_DRIVER, {})
+
+        # Check if driver_config is a string (old syntax)
+        if isinstance(driver_config, str):
+            driver_instance_str = driver_config
+            prob.driver = self._om_eval(driver_instance_str)
+        else:
+            # Use new syntax
+            # Set the driver instance
+            driver_instance = driver_config.get("instance")
+            if driver_instance:
+                driver_instance = self._om_eval(driver_instance)
+                prob.driver = driver_instance
+            else:
+                # Raise error if no driver instance is provided
+                raise FASTConfigurationBaseKeyBuildingError(
+                    ValueError("No driver instance provided in configuration file"),
+                    KEY_DRIVER,
+                )
```

Comment:
I think this code can be deleted, because the JSON schema already does this check.

Reply:
Corrected in [3c7400f](https://github.com/fast-aircraft-design/FAST-OAD/pull/559/commits/3c7400f9d41f4d25e8c0a2c25ea1db12d5fa8941)